src/hooks/useFlipCameraToggle/useFlipCameraToggle.tsx

```diff
@@ -1,16 +1,14 @@
-import React, { useCallback, useEffect, useState } from 'react';
-import { Button } from '@material-ui/core';
-import { DEFAULT_VIDEO_CONSTRAINTS } from '../../../constants';
-import FlipCameraIcon from './FlipCameraIcon';
+import { useCallback, useEffect, useState } from 'react';
+import { DEFAULT_VIDEO_CONSTRAINTS } from '../../constants';
 import { LocalVideoTrack } from 'twilio-video';
-import useDevices from '../../../hooks/useDevices/useDevices';
-import useMediaStreamTrack from '../../../hooks/useMediaStreamTrack/useMediaStreamTrack';
-import useVideoContext from '../../../hooks/useVideoContext/useVideoContext';
+import useDevices from '../useDevices/useDevices';
+import useMediaStreamTrack from '../useMediaStreamTrack/useMediaStreamTrack';
+import useVideoContext from '../useVideoContext/useVideoContext';
 
-export default function FlipCameraButton() {
+export default function useFlipCameraToggle() {
   const { localTracks } = useVideoContext();
   const [supportsFacingMode, setSupportsFacingMode] = useState<Boolean | null>(null);
```

Comment:
Looks like I wrote this line, but looking at it again I'm not sure why `supportsFacingMode` is of type `Boolean | null`. I think it would make more sense if it was just `Boolean`. 

What do you think about this change? It doesn't change the function of this hook, does it?

```suggestion
  const [supportsFacingMode, setSupportsFacingMode] = useState(false);
```

Would also have to change line 22 to: `    if (currentFacingMode && supportsFacingMode === false) {`

This change results in `flipCameraSupported` being of type `Boolean` instead of `Boolean | null`, which makes more sense to me.

Reply:
that makes sense to me too. i googled around a little bit about `MediaStreamTrack` since it was new to me and was able to access this property locally by exploring `twilioRoom.localParticipant.tracks`. I confirmed that `facingMode` is not included in the object returned by `mediaStreamTrack.getSettings()` when on desktop, which means it's not supported so I agree that we should use `false.` TLDR: I agree and made the update 🙂